.travis.yml

```diff
@@ -21,9 +21,11 @@ install:
 - pip3 install .[dev]
 script:
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then pytest --cov-config=.coveragerc --cov=SwSpotify
-  tests/test_spotify.py::LinuxTests; fi
+  tests/test_spotify.py::LinuxTests; pytest --cov-config=.coveragerc --cov=SwSpotify
```

Comment:
If you're executing WebTests in all cases, why do them in an if fi;? You can call them in their own separate line